pom.xml

```diff
@@ -2035,6 +2041,8 @@
 
         <!-- REST API feature -->
         <identity.api.dispatcher.version>0.1.12</identity.api.dispatcher.version>
+        <!--Rest API test -->
+        <rest-assured.version>3.0.0</rest-assured.version>
```

Comment:
Shall we use the property as `rest.assured.version` to be consistent with the others.